.mailmap

```diff
@@ -0,0 +1,31 @@
+Andrew Myers <atmyers@lbl.gov>              Andrew Myers <atmyers2@gmail.com>
+Andrew Myers <atmyers@lbl.gov>              atmyers <atmyers2@gmail.com>
+Aurore Blelly <ablelly@lbl.gov>             ablelly <aurore.blelly@ensta-paristech.fr>
+Aurore Blelly <ablelly@lbl.gov>             ablelly <48810880+ablelly@users.noreply.github.com>
+Axel Huebl <axelhuebl@lbl.gov>              Axel Huebl <axel.huebl@plasma.ninja>
+David Grote <dpgrote@lbl.gov>               grote <dpgrote@lbl.gov>
+David Grote <dpgrote@lbl.gov>               Dave <grote1@llnl.gov>
+David Grote <dpgrote@lbl.gov>               Dave Grote <grote1@llnl.gov>
+David Grote <dpgrote@lbl.gov>               Grote <grote1@n9459722.llnl.gov>
+Edoardo Zoni <ezoni@lbl.gov>                Edoardo Zoni <edoardo.zoni@ipp.mpg.de>
+Ligia Diana Amorim <ldianaamorim@lbl.gov>   L. Diana Amorim <LDianaAmorim@lbl.gov>
+Ligia Diana Amorim <ldianaamorim@lbl.gov>   Diana Amorim <diana@henrivincenti.dhcp.lbl.gov>
+Ligia Diana Amorim <ldianaamorim@lbl.gov>   Ligia Diana Amorim <ligiada@cori01.nersc.gov>
+Ligia Diana Amorim <ldianaamorim@lbl.gov>   Ligia Diana Amorim <ligiada@cori11.nersc.gov>
+Luca Fedeli <luca.fedeli@cea.fr>            Luca Fedeli <luca.fedeli.88@gmail.com>
```

Comment:
Hi! Thanks for this PR!

Using `git log --format='%aN <%aE>' | sort -u` in the WarpX folder I got this list of aliases:
`
lucafedeli88 <luca.fedeli@cea.fr>
Luca Fedeli <luca@DEB.station>
Luca Fedeli <luca.fedeli.88@gmail.com>
Luca Fedeli <luca.fedeli@cea.fr>
Luca Fedeli <luca.fedeli@for.unipi.it>
Luca <luca@localhost.localdomain>
`
Only two of them are currently included in the `.mailmap`. Should the other be included as well?

Reply:
Thanks a lot, yes will update!